### PR TITLE
chore: add missing GITHUB_TOKEN values...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: add missing GITHUB_TOKEN values before running ./build.sh (#19986)

Change-Id: I8f593a2f57439a283874756c518eaa14620e20dd
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by benoitf** [comment]
Afaik we don't need it in github actions but in Jenkins CRW instance

Jobs in github actions don't have any issue

**Comment by nickboldt** [comment]
> Afaik we don't need it in github actions but in Jenkins CRW instance
> Jobs in github actions don't have any issue

In that case see 

* https://issues.redhat.com/browse/CRW-1931
* https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/626/diffs

File: .github/workflows/build-publish-next.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
     - name: Build and publish images
       run: |
         docker image prune -a -f
+        export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
```
Comment (azatsarynnyy) [inline .github/workflows/build-publish-next.yml:42]: `GITHUB_TOKEN` secret is created automatically
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Comment (nickboldt) [reply .github/workflows/build-publish-next.yml:42]: yes, but only when the secret == secrets.GITHUB_TOKEN.

Here, we're using a different secret name, secrets.CHE_BOT_GITHUB_TOKEN.

But if you don't think we need this, you can close the PR.

Comment (benoitf) [reply .github/workflows/build-publish-next.yml:42]: it's better to use the generated token as we have a fresh one for each run (while we could decrease quite a lot the one from che-bot)

**Comment by benoitf** [comment]
closing as it is not required

**Pull request closed without merging**